src-web/components/kappnav/ComponentView.jsx

```diff
@@ -44,23 +108,45 @@ class ComponentView extends React.Component {
       sortDirection: SORT_DIRECTION_ASCENDING,
       pageSize: PAGE_SIZES.DEFAULT,
       pageNumber: 1,
-      search: undefined
+      search: undefined,
+      resourceType : this.getResourceType(),
+      name : this.getName()
     };
 
     this.fetchData = this.fetchData.bind(this);
     this.toggleExpandCollapse = this.toggleExpandCollapse.bind(this)
   }
 
   render() {
-    const resourceType = this.props.resourceType
+    var paths = this.getTokenizedURLPath()
+    let title = msgs.get('page.applicationView.title') // default = application
+    let titleUrl = CONTEXT_PATH + '/applications' // default = application
+    let resourceType = this.state.resourceType
     const resourceData = getResourceData(resourceType)
 
+    if (paths[1] === 'was-nd-cells') {
+      // Change title to be WAS ND Cells
+      title = msgs.get('page.wasNdCellsView.title')
+      titleUrl = CONTEXT_PATH + '/was-nd-cells'
+    } else if (paths[1] === 'liberty-collectives') {
+      // Change title to be Liberty Collectives
+      title = msgs.get('page.libertyCollectiveView.title')
+      titleUrl = CONTEXT_PATH + '/liberty-collectives'
+    }
```

Comment:
I do not think we should have references to `was` or `liberty` specific resources in kAppNav